source/Engine.cpp

```diff
@@ -1205,6 +1205,62 @@ void Engine::BreakTargeting(const Government *gov)
 
 
 
+unsigned Engine::FleetPlacementLimit(const LimitedEvents<Fleet> &fleet, unsigned frames, bool requireGovernment)
```

Comment:
in the header this function is defined after DoGrudge(), move it either in the header or in the cpp